packages/wp-story-editor/src/components/mediaUpload/mediaPicker/useMediaPicker.js

```diff
@@ -92,7 +99,58 @@ function useMediaPicker({
     } catch (e) {
       // Silence.
     }
-  }, [updateMedia]);
+  }, [onUpload, updateMedia]);
+
+  useEffect(() => {
+    const wsDetails = window.wp.media.view.Attachment.Details?.extend({
+      deleteAttachment: function (event) {
+        event.preventDefault();
+
+        this.getFocusableElements();
+
+        if (window.confirm(window.wp.media.view.l10n.warnDelete)) {
+          this.model.destroy({
+            wait: true,
+            error: function () {
+              window.alert(window.wp.media.view.l10n.errorDeleting);
+            },
+          });
+
+          this.moveFocus();
+        }
```

Comment:
Where is this copied from? Instead of copying, can we just call the original `deleteAttachment` function or something? Ditto for `trashAttachment` below

Reply:
Fixed.